2200091/10505. 소득 불균형.py

```diff
@@ -0,0 +1,24 @@
+import sys
+input = sys.stdin.readline
+
+# 첫 번째 테스트 케이스 수를 저장하는 변수
+test = int(input())
+
+for i in range(test):
+    # 두 번째 테스트 케이스 수를 저장하는 변수
+    num = int(input())
+    # 평균 이하인 수를 저장하는 변수
+    cnt = 0
+
+    # num개의 수를 입력받아 리스트로 저장
+    income = list(map(int, input().split()))
+
+    # 리스트의 평균을 계산하여 변수에 저장
+    mean = sum(income) // num
```

Comment:
좋은 풀이네요!
```suggestion
    mean = sum(income) / num
```
정확한 평균 값을 구하려면 `//`가 아닌 `/` 연산을 사용하시는 게 맞습니다! 😄